slsSupportLib/include/sls/versionAPI.h

```diff
@@ -4,11 +4,11 @@
 #define RELEASE      "7.0.0.rc1"
```

Comment:
?

Reply:
Before it used to be called branch. In the end, its used in "versions" command as "packageversion"